testutils/src/test/scala/za/co/absa/ConfigSuite/testutils/ConfigSuite.scala

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.ConfigSuite.testutils
+
+import org.scalatest.FunSuite
+import za.co.absa.testutils.CmdConfig
+
+class ConfigSuite extends FunSuite {
+
+  private val rawFormat = "parquet"
```

Comment:
You only use this once, everywhere else it's just a String literal inline. I'd skip this and keep it consistent. Or if you prefer extract all formats as fields and call them `private val csvForamt = "cvs"`, etc.